test/e2e/tests/settings/address-book.spec.js

```diff
@@ -70,6 +70,61 @@ describe('Address Book', function () {
       },
     );
   });
+
+  it('Adds a new contact to the address book', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver }) => {
+        await unlockWallet(driver);
+        await openMenuSafe(driver);
+
+        await driver.clickElement({ text: 'Settings', tag: 'div' });
+        await driver.clickElement({ text: 'Contacts', tag: 'div' });
+
+        await driver.clickElement('.address-book__link');
+
+        const inputUsername = await driver.findElement('#nickname');
+        await inputUsername.fill('#nickname', 'Test User');
```

Comment:
Could you please update this to the below suggestion to avoid race condition

```suggestion
        await driver.fill('#nickname','Test User');
```